Bottom: 664d05fcc84bb30f9d5286e8310a0aaed1ec6577
Top:    7c7b66cabaecae46d374eabe0d462d38d0d4aa89
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-13 22:31:33 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/mpc5200_dma.c b/sound/soc/fsl/mpc5200_dma.c
index 5ac2693..2c42f29 100644
--- a/sound/soc/fsl/mpc5200_dma.c
+++ b/sound/soc/fsl/mpc5200_dma.c
@@ -561,7 +561,7 @@ int mpc5200_audio_dma_create(struct of_device *op, struct snd_soc_dai *template,
 	psc_dma->dev = &op->dev;
 	psc_dma->playback.psc_dma = psc_dma;
 	psc_dma->capture.psc_dma = psc_dma;
-	snprintf(psc_dma->name, sizeof psc_dma->name, "PSC%u AC97", psc_id+1);
+	snprintf(psc_dma->name, sizeof psc_dma->name, "PSC%u", psc_id+1);
 
 	/* Fill out the CPU DAI structure */
 	nDAI = min(tsize, SOC_OF_SIMPLE_MAX_DAI);
diff --git a/sound/soc/fsl/mpc5200_psc_i2s.c b/sound/soc/fsl/mpc5200_psc_i2s.c
index 513d68d..3f76f6a 100644
--- a/sound/soc/fsl/mpc5200_psc_i2s.c
+++ b/sound/soc/fsl/mpc5200_psc_i2s.c
@@ -144,6 +144,7 @@ static struct snd_soc_dai_ops psc_i2s_dai_ops = {
 };
 
 static struct snd_soc_dai psc_i2s_dai_template = {
+	.name   = "%s I2S",
 	.playback = {
 		.channels_min = 2,
 		.channels_max = 2,
